packages/framework/tests/Test/Codeception/ActorInterface.php

```diff
@@ -0,0 +1,963 @@
+<?php
```

Comment:
wow, it was really hard to review the whole class :smile: 
And it seems that it will be quite hard to maintain this class when Codeception is upgraded in the future - how can we assure the interface is up to date with possible changes in the generated `AcceptanceTesterActions` trait and with the other modules?

I do not like we need this cumbersome interface only because of the fact that `AcceptanceTester` class is in `project-base`. Could this be solved by https://github.com/shopsys/shopsys/issues/1246?

Reply:
I might be helped in #1246, but I don't really know for sure. This will at least give us some way to see whether some methods, that were previously supported, get removed. And of course, if we in future don't need a particular `Actor` method we don't have to add it to the interface. Adding methods to the interface would be considered a BC break as per our promise.

For me, the question was "Which methods should I add to the interface?" I could have chosen only the methods that I actually use in the helper classes but that seemed pretty shortsighted to me. The other obvious choice was all of them (which I've chosen). There could be some arbitrary subset of the supported methods but I couldn't find a reasonable dividing line...

Reply:
ok, thanks for the answer :+1: